src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -328,124 +328,26 @@ public static float interpolateRadians(float factor, float start, float end) {
 		return normalizeRad(val);
 	}
 
+	/** rebuilds the `currentBoneInfo` list */
```

Comment:
That's not how you format java docs, please add some line breaks!

Reply:
I thought this was viable based on prior use of IntelliJ, but I believe you are correct: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#examples

Reply:
fixed